lib/src/nyxx.dart

```diff
@@ -202,22 +200,30 @@ class NyxxRest extends INyxxRest {
 
   @override
   Future<void> connect({bool propagateReady = true}) async {
+    _logger.config([
+      'Token: $token',
```

Comment:
Is it a good idea to log the token to stdout?

Reply:
It's already logged in HTTP request logging, so it doesn't really matter. What would be a good idea is to add an option to `Logging` to censor the token in the output, so I'll do that.